chia/server/node_discovery.py

```diff
@@ -212,7 +206,9 @@ async def on_connect(peer: WSChiaConnection) -> None:
             msg = make_msg(ProtocolMessageTypes.request_peers_introducer, RequestPeersIntroducer())
             await peer.send_message(msg)
 
-        await self.server.start_client(self.introducer_info, on_connect)
+        await self.server.start_client(
+            PeerInfo(get_host_addr(self.introducer_info.host, prefer_ipv6=False), self.introducer_info.port), on_connect
```

Comment:
this is still a blocking call to `get_host_addr()`. It looks like this is just shifting the problem around a bit without solving the underlying issue. I would expect this call to be async.

Reply:
See https://github.com/Chia-Network/chia-blockchain/pull/14999